src/scripts/ci/travis.yml

```diff
@@ -88,6 +94,12 @@ matrix:
     - os: linux
       compiler: clang
       env: BUILD_MODE="cross-arm64"
+    # No docker build on osx
+    - os: osx
+      env: BUILD_MODE="docker-android"
+    # Nor for gcc's
+    - compiler: gcc
+      env: BUILD_MODE="docker-android"
```

Comment:
I think these are not being excluded from the build because now env contains multiple values and Travis wants to see a complete match of the env for an exclusion.

I think the best fix is

ANDROID_ARCH=arm ANDROID_TOOLCHAIN_SUF=eabi BUILD_MODE="docker-android" -> BUILD_MODE="android-arm7"

ANDROID_ARCH=arm64 BUILD_MODE="docker-android" -> BUILD_MODE="android-aarch64"

Then exclusions work.

However, we now need to get the relevant env vars down to the docker script. I'm going to push a change adding that to the CI build script in this branch.